client/js/util.js

```diff
@@ -2,6 +2,7 @@ const KEYS = {
   TAB: 9,
   ENTER: 13,
   ESCAPE: 27,
+  SPACEBAR: 32,
```

Comment:
Ah thanks. Was a hangover from before I was using button and was replicating the spacebar keydown event. Will remove.